lib/get_player_input.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'tic_tac_toe.rb'
+
+def get_player_input(input, player, board=Array.new(3){Array.new(3, "_")})
+    puts "Player #{player}, enter your placeholder in format num1, num2"
+    input = input.gets.chomp
+
+    row = input.split(",")[0].to_i
+    col = input.split(",")[1].to_i
+
+    if board[row][col] == "_"
```

Comment:
Note how there are now two places that need to know that `"_"` is an unoccupied space on the board.   We will get back to this idea soon when we explore object designs, so that there's only one piece of code that needs to know that.

No change is necessary right now - I'm just mentioning something to remember soon.